apiplugin/file/filehost.go

```diff
@@ -85,6 +86,20 @@ func (*filePlugin) Init(ctx context.Context, e eng.Engine) bool {
 	return true
 }
 
+func (fi *fileInfo) Read(p []byte) (n int, err error) {
+	for i, b := range []byte(fi.content) {
+		if i >= len(p) {
+			fi.lastAccessTime = time.Now()
```

Comment:
Are you maintaining the lastAccessTime because you are keeping everything in memory?

Reply:
exactly